packages/frontend/src/renderer/components/CreateJoinCommunity/JoinCommunity/JoinCommunity.test.tsx

```diff
@@ -56,7 +54,7 @@ describe('join community', () => {
     expect(createCommunityTitle).toBeVisible()
   })
 
-  it('user goes form joning community to username registration, then comes back', async () => {
+  it.skip('user goes form joning community to username registration, then comes back', async () => {
```

Comment:
This test is skipped because circumstances has changed. It should be restored after introducing multiple communities